### DISCUSSION TITLE
Clarification from the sponsors page.

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#4730] [Q&A] [answered]
I wanted some clarification on the sponsors page (which looks a little different when the $10 tier was the lowest)

```
$15 a month
The Individual

Choose for non-commercial use (1-2 sites)

You love Material for MkDocs and use it for your Open Source project or personal blog. 
You read about all the awesome features in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/), which are first exclusively released to sponsors, and don't want to miss out.
```

> Choose for non-commercial use (1-2 sites)

Does that mean the insiders is now limited to 2 sites for a $15 tier (& can't be used more than that, regardless of open or closed source) ?

**Comment by squidfunk** [comment, marked as answer]
Thanks for asking! The intention behind this is user group segmentation. You should choose this tier if you use Insiders for a handful of non-commercial installations. If you build _many_ non-commercial sites, please go for the $35 tier, which doesn't have this limit. The reason is that we need to segment specific use cases, and the number of sites is currently the only value metric we have, since everybody can fork the project to a private account and then share access with an arbitrary number of users. We don't have a "per seat" limit, albeit that would be a better value metric. Maybe in the future, when this is possible, we'll change to that.

Please note that the whole sponsorship approach is a hell of a big experiment I'm currently conducting. I'm changing stuff here and there to see how it works, so any feedback is appreciated. Thus, thanks for sharing! I'd be curious to hear your thoughts on this. I'm also open to discussing alternative segmentation approaches. It's all work in progress.

**Reply by squidfunk** [reply]
... and to expand on why "number of sites" is the current value metric: building more sites means more surface where things can go wrong, or where you have questions that you ask here that we need to answer. See the issue tracker and discussion board to learn how responsive we are, answering questions every single day. Thus, it's more work for us. It's a simple calculation. If you run into a bug when using Insiders (or Material for MkDocs), you can report it here and we usually fix it within a few days. Compare that to other Open Source projects which have hundreds of open issues.

We're working very hard to keep it under 10 – 20 max 😊

**Reply by UltraInstinct05** [reply]
Sorry for the late reply. I had previously sponsored for many months (under the $10 tier, before it changed to $15). My sponsorship got canceled because I manually pay through GitHub payment page every month (did not find time from `$DAYJOB` to setup a way to have a standing instruction to auto pay it) and forgot to do it within time recently.

Was thinking of re sponsoring for $15 but the "no of usage" site limitation is now deterring me. I (like many else probably) would like to use material for mkdocs for non commercial or open source sites (even open source "books") so the restriction is kind of a stopper (I know there is no restriction or way to check, but if you use something, you need to respect the conditions of usage). I do understand the reason behind it (for people who choose a tier, then private fork it and essentially can share with anyone else) but just my thoughts.

**Reply by squidfunk** [reply]
Thanks for your thoughts and thanks for sponsoring! I understand your concerns and can already say that I'm actively working towards a solution that will allow to limit it to accounts/users with no limitations on how many sites are built. It is far from being trivial, since GitHub Sponsors has very limited ability to interact with sponsors, but as said, the sponsor experience will be much, much improved next year. You can expect a solution in the coming months, so the # of sites limitations will then be lifted again.
